### ISSUE TITLE
it no work in new version of ultrakill help

### ISSUE THREAD

**Issue by kevbealive** [Temperz87/ultra-mod-manager#34]
engineer engineering

**Comment by ERENPROO** [comment]
@Temperz87 answer we need you :/

**Comment by Temperz87** [comment]
It'll happen at some point don't worry, I'm just doing a bunch of rewrites and need to decouple some stuff and shove it into another mod.
In other news, as much as I would love to spam gifs on this issue, this project is on my Resume so I unfortunately can't.

**Comment by CursedSFMS** [comment]
Thank you so much Temperz soon i will be able to 360 noscope the Earthmover

**Comment by Temperz87** [comment]
You guys know you can use mods without UMM right?

**Comment by PlanetSat0rn** [comment]
dude, most of the mods i use are on UMM

**Comment by CursedSFMS** [comment]
yeah mods like WafflesWeapons and such dont work for some reason without UMM

**Comment by PlanetSat0rn** [comment]
wheres the fix

**Comment by updateplsNOW** [comment]
where is update

**Comment by PlanetSat0rn** [comment]
its official

the creator killed umm

**Comment by CursedSFMS** [comment]
Wait WHAT

**Comment by Temperz87** [comment]
Can you guys not spam this thread?
No, UMM is not dead.
Yes, it will probably reappear in a new state.
Yes, it won't be updated for possibly months.

**Comment by BattleMedicTF2** [comment]
My bad my bad

**Comment by TeamDoodz** [comment]
Why don't john temperz just press the update button?
